tests/test_mesh.py

```diff
@@ -1,17 +1,64 @@
-import build123d as bd
+"""Meshing tests."""
+
+import filecmp
+from pathlib import Path
+
+import gmsh
 import pytest
 import stellarmesh as sm
 
+from .test_geometry import (
+    geom_bd_imprintedboxes,
+    geom_bd_layered_torus,
+    geom_bd_nestedspheres,
+    geom_bd_sphere,
+)
+
+
+@pytest.mark.parametrize(
+    "model_name",
+    [
+        geom_bd_layered_torus.__name__,
+        geom_bd_nestedspheres.__name__,
+        geom_bd_sphere.__name__,
+        geom_bd_imprintedboxes.__name__,
+    ],
+)
+def test_mesh_output(model_name: str, request: pytest.FixtureRequest, tmp_path: Path):
+    """Ensure that generated meshes match benchmark .msh files."""
+    model = request.getfixturevalue(model_name)
+    msh_name = f"mesh_{model_name}.msh"
+    mesh = sm.SurfaceMesh.from_geometry(
+        model,
+        sm.OCCSurfaceOptions(0.1, 10),
+    )
```

Comment:
Some thoughts -- this is a pretty clean approach that avoids needing to have a separate argument indicating the meshing backend, but I would just caution that it may not immediately be obvious to people so make sure that the documentation makes clear how it works.